private/buf/cmd/buf/command/alpha/registry/token/tokenlist/tokenlist.go

```diff
@@ -86,6 +89,12 @@ func (f *flags) Bind(flagSet *pflag.FlagSet) {
 		bufprint.FormatText.String(),
 		fmt.Sprintf(`The output format to use. Must be one of %s`, bufprint.AllFormatsString),
 	)
+	flagSet.Uint32Var(
+		&f.OrderedColumn,
+		orderedColumnFlagName,
+		1,
+		`The column by which to sort results.`,
```

Comment:
What does this look like on the command line? Just `--ordered-column 1`? Can we make this use a string representation, if we need to add it at all?